src/js/post-911-gib-status/utils/helpers.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+export function formatPercent(percent) {
+  let validPercent = undefined;
+
+  if (!isNaN(parseInt(percent, 10))) {
+    validPercent = `${Math.round(percent)}%`;
+  }
+
+  return validPercent;
+}
+
+export function formatVAFileNumber(n) {
+  const number = n || '';
+  const lengthOfXString = number.length > 4 ? number.length - 4 : 0;
+
+  return number.replace(number.substring(0, lengthOfXString),
+                        `${'x'.repeat(lengthOfXString)}-`);
+}
+
+export function formatMonthDayFields(field) {
+  let displayValue;
+  if (field) {
+    displayValue = `${field.months} months, ${field.days} days`;
+  } else {
+    displayValue = 'unavailable';
+  }
+  return displayValue;
+}
+
+export function systemDownMessage() {
+  return (
+    <div className="row">
+      <div className="small-12 columns">
+        <div className="react-container">
+          <h3>Sorry, our system is temporarily down while we fix a few things. Please try again later.</h3>
+          <a href="/"><button>Go Back to Vets.gov</button></a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function chapter33InfoUnavailableWarning() {
```

Comment:
Sophia says to replace this with a generic no-records warning like this one: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/disability-benefits/components/MviRecordsUnavailable.jsx